blade-graphics/src/gles/command.rs

```diff
@@ -806,6 +810,65 @@ impl super::Command {
                     None,
                 );
             }
+
+            Self::BlitFramebuffer { from, to } => {
+                /*
+                    TODO(ErikWDev): Validate
+                */
+
+                let target_from = match from.inner {
+                    super::TextureInner::Renderbuffer { raw } => raw,
+                    _ => panic!("Unsupported resolve between non-renderbuffers"),
+                };
+                let target_to = match to.inner {
+                    super::TextureInner::Renderbuffer { raw } => raw,
+                    _ => panic!("Unsupported resolve between non-renderbuffers"),
+                };
+
+                let framebuf_from = gl.create_framebuffer().unwrap();
+                let framebuf_to = gl.create_framebuffer().unwrap();
+
+                gl.bind_framebuffer(glow::READ_FRAMEBUFFER, Some(framebuf_from));
+                gl.framebuffer_renderbuffer(
+                    glow::READ_FRAMEBUFFER,
+                    glow::COLOR_ATTACHMENT0, // NOTE: Assuming color attachment
+                    glow::RENDERBUFFER,
+                    Some(target_from),
+                );
+
+                gl.bind_framebuffer(glow::DRAW_FRAMEBUFFER, Some(framebuf_to));
+                gl.framebuffer_renderbuffer(
+                    glow::DRAW_FRAMEBUFFER,
+                    glow::COLOR_ATTACHMENT0, // NOTE: Assuming color attachment
+                    glow::RENDERBUFFER,
+                    Some(target_to),
+                );
+                assert_eq!(
```

Comment:
let's switch this to `debug_assert_eq!` - no need for the extra GL call in release